<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SuperPixelSegmentationGUI</class>
 <widget class="QMainWindow" name="SuperPixelSegmentationGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Super Pixel Segmentation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label1">
          <property name="text">
           <string>K: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="FloatSlider" name="sldK">

          <property name="whatsThis">
           <string>The float slider widget allows the user to select a float value.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Sigma:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="FloatSlider" name="sldSigma">
          <property name="whatsThis">
           <string>The float slider widget allows the user to select a float value.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>MinSize:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="LabeledSlider" name="sldMinSize">
          <property name="whatsThis">
           <string>The slider widget displays the min, max, and selected values.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btnSegment">
        <property name="text">
         <string>Segment</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkShowInput">
        <property name="text">
         <string>Input</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkShowSegments">
        <property name="text">
         <string>Segments</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenImage"/>
    <addaction name="actionSaveResult"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenImage">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionSaveResult">
   <property name="text">
    <string>Save Result</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FloatSlider</class>
   <extends>QWidget</extends>
   <header>FloatSlider.h</header>
  </customwidget>
  <customwidget>
   <class>LabeledSlider</class>
   <extends>QWidget</extends>
   <header>LabeledSlider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
